### PR TITLE
cdc: no hint_min_ts for cdc incremental scan

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### What problem does this PR solve?

Issue Number: cc https://github.com/pingcap/ticdc/issues/3265

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

1. create changefeed
2. go-tpcc prepare
3. go-tpcc run
4. kill tikv and kill cdc
5. sync-diff check

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by overvenus — APPROVED** [review]
LGTM

**Comment by overvenus** [comment]
It has been tested more than 12 hours, everything looks good.

File: components/cdc/src/endpoint.rs [hunk 1]
```diff
@@ -1255,7 +1255,7 @@ impl Initializer {
         let mut scanner = ScannerBuilder::new(snap, current)
             .fill_cache(false)
             .range(None, None)
-            .hint_min_ts(Some(self.checkpoint_ts))
+            // .hint_min_ts(Some(self.checkpoint_ts))
```
Comment (liuzix) [inline components/cdc/src/endpoint.rs:1258]: Please add a `FIXME` here

**Comment by maxshuang** [comment]
I also wonder why all the inconsistency is about "workload.new_order".  Expect an explanation after investigation.

**Comment by amyangfei** [comment]
> I also wonder why all the inconsistency is about "workload.new_order". Expect an explanation after investigation.

Does other table have delete SQL?

**Comment by overvenus** [comment]
It has been fixed by https://github.com/tikv/tikv/pull/11318 .

**Pull request closed without merging**